sql/branch.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+CREATE TABLE feedback
+( id            serial      PRIMARY KEY
+, participant   bigint      NOT NULL
```

Comment:
This column needs to be indexed, otherwise the `/admin/users` page will be inefficient and possibly slow. Since we only need one feedback per account, I suggest dropping the `id` column and making the `participant` column the primary key.